pkg/cmd/projectconfig/import.go

```diff
@@ -0,0 +1,243 @@
+	"github.com/daytonaio/daytona/pkg/views"
+	views_util "github.com/daytonaio/daytona/pkg/views/util"
+	"github.com/daytonaio/daytona/pkg/views/workspace/create"
+	"github.com/daytonaio/daytona/pkg/views/workspace/selection"
+	"github.com/spf13/cobra"
+)
+
+var filePath string
+
+var projectConfigImportCmd = &cobra.Command{
+	Use:     "import",
+	Aliases: []string{"imp"},
+	Short:   "Import project config from JSON",
+	Args:    cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var inputText string
+		ctx := context.Background()
+
+		apiClient, err := apiclient_util.GetApiClient(nil)
+		if err != nil {
+			return err
+		}
+
+		projectConfigList, res, err := apiClient.ProjectConfigAPI.ListProjectConfigs(ctx).Execute()
+		if err != nil {
+			return apiclient_util.HandleErrorResponse(res, err)
+		}
+
+		if filePath != "" {
+			if filePath == "-" {
+				inputBytes, err := io.ReadAll(os.Stdin)
+				if err != nil {
+					return fmt.Errorf("error reading stdin: %v", err)
+				}
+				inputText = string(inputBytes)
+			} else {
+				inputBytes, err := os.ReadFile(filePath)
+				if err != nil {
+					return fmt.Errorf("error reading file: %v", err)
+				}
+				inputText = string(inputBytes)
+			}
+
+		} else {
+			form := huh.NewForm(
+				huh.NewGroup(
+					huh.NewText().
+						Title("Import Project-Config").
+						Description("Enter Project-Config as a JSON or an array of JSON objects").
+						CharLimit(-1).
+						Value(&inputText),
+				),
+			).WithTheme(views.GetCustomTheme()).WithHeight(20)
+			err = form.Run()
+			if err != nil {
+				return err
+			}
+		}
+
+		var config apiclient.ProjectConfig
+		err = json.Unmarshal([]byte(inputText), &config)
+		if err == nil {
+			err = importProjectConfig(ctx, apiClient, config, &projectConfigList)
+			if err != nil {
+				return err
+			}
+		} else {
+			var configs []apiclient.ProjectConfig
+			err = json.Unmarshal([]byte(inputText), &configs)
+			if err != nil {
+				return fmt.Errorf("invalid JSON input: %v", err)
+			}
+
+			for _, config := range configs {
+				err = importProjectConfig(ctx, apiClient, config, &projectConfigList)
+				if err != nil {
+					return err
+				}
+			}
+		}
+		return nil
+	},
+}
+
+func init() {
+	projectConfigImportCmd.Flags().StringVarP(&filePath, "file", "f", "", "Import project config from a JSON file. Use '-' to read from stdin.")
+}
+
+func isProjectConfigAlreadyExists(configName string, projectConfigList *[]apiclient.ProjectConfig) bool {
+	for _, projectConfig := range *projectConfigList {
+		if projectConfig.Name == configName {
+			return true
+		}
+	}
+	return false
+}
+
+func importProjectConfig(ctx context.Context, apiClient *apiclient.APIClient, config apiclient.ProjectConfig, projectConfigList *[]apiclient.ProjectConfig) error {
+	if isProjectConfigAlreadyExists(config.Name, projectConfigList) {
+		views.RenderInfoMessage(fmt.Sprintf("Project config already present with name \"%s\"", config.Name))
```

Comment:
We should treat this as an error because the project config "failed" to import

Reply:
Done!